handler.js

```diff
@@ -1,20 +1,24 @@
+// @ts-check
```

Comment:
Is a `tsconfig.json` file needed to make this work?

Reply:
Done – good catch.